transformations/numeric_to_word/test.json

```diff
@@ -0,0 +1 @@
ss launches defence lawyers defending former worldcom chief bernie ebbers against a battery of fraud charges have called a company whistleblower as their first witness."}, "outputs": [{"sentence": "one thousand, eight hundred and thirty-three ,worldcom ex-boss launches defence lawyers defending former worldcom chief bernie ebbers against a battery of fraud charges have called a company whistleblower as their first witness."}]}, {"class": "NumericToWord", "inputs": {"sentence": "other pre-electronic gadgets in the top 100 include the sextant from 1731 (59th position)  the marine chronometer from 1761 (42nd position) and the kodak brownie camera from 1900 (28th position)"}, "outputs": [{"sentence": "other pre-electronic gadgets in the top one hundred include the sextant from seventeen thirty-one ( fifty-ninth position) the marine chronometer from seventeen sixty-one ( forty-second position) and the kodak brownie camera from one thousand nine hundred ( twenty-eighth position)"}]}, {"class": "NumericToWord", "inputs": {"sentence": "australian open champion williams survived an early scare to beat russia s elena bovina 1-6 6-1 6-4. world number one lindsay davenport and anastasia myskina also progressed"}, "outputs": [{"sentence": "australian open champion williams survived an early scare to beat russia s elena bovina one to six six to one six to four . world number one lindsay davenport and anastasia myskina also progressed"}]}, {"class": "NumericToWord", "inputs": {"sentence": "odds on kenzie s victory have been slashed from 4/6 to 1/2  although bookmakers have said they are not ruling out a late rally from former happy mondays star bez"}, "outputs": [{"sentence": "odds on kenzie s victory have been slashed from four over six to a half although bookmakers have said they are not ruling out a late rally from former happy mondays star bez"}]}, {"class": "NumericToWord", "inputs": {"sentence": "moya fights back for indian title carlos moya became the first man to successfully defend the chennai open title by beating four-times finalist paradorn srichaphan 3-6 6-4 7-6 (7/5)"}, "outputs": [{"sentence": "moya fights back for indian title carlos moya became the first man to successfully defend the chennai open title by beating four-times finalist paradorn srichaphan three to six six to four seven to six (seven over five)"}]}, {"class": "NumericToWord", "inputs": {"sentence": "last week  mci agreed to a buyout by verizon communications in a deal valued at $6.75bn.,business"}, "outputs": [{"sentence": "last week mci agreed to a buyout by verizon communications in a deal valued at six point seven five bn. dollar business"}]}, {"class": "NumericToWord", "inputs": {"sentence": "the sequel took $44.7m (\u00a323.2m) between 24 and 26 december  according to studio estimates"}, "outputs": [{"sentence": "the sequel took forty-four point seven m dollar (twenty-three point two m pound) between twenty-four and twenty-sixth of december according to studio estimates"}]}]}
```

Comment:
Also, please format the json - If you are using pycharm - you can -select all and do CTRL+ALT+SHIFT+L :)

Reply:
Will try and fix, thank you @kaustubhdhole